Dockerfile

```diff
@@ -5,14 +5,15 @@ COPY . /orchestrator-relayer
 WORKDIR /orchestrator-relayer
 RUN make build
 
-# stage 2
-FROM alpine:3.17.3
+# final image
+FROM alpine:3.17.2
```

Comment:
Why not keep `3.17.3`?

Reply:
actually, this is the one present in the e2e Dockerfile that's why I kept it, I don't have strong considerations around minor versions in alpine.

Reply:
let's leave it for now then, it's cool